mailpoet/lib/Mailer/Methods/ErrorMappers/MailPoetMapper.php

```diff
@@ -196,10 +196,32 @@ private function getEmailVolumeLimitReachedMessage(): string {
     return "{$message}<br/>";
   }
 
+  private function getPendingApprovalMessage(): string {
+    $message = __("Your subscription is currently [link]pending approval[/link].You'll soon be able to send once our team reviews your account. In the meantime, you can send previews to your authorized emails.", 'mailpoet');
+    $message = Helpers::replaceLinkTags(
+      $message,
+      'https://kb.mailpoet.com/article/350-pending-approval-subscription',
+      [
+        'target' => '_blank',
+        'rel' => 'noopener noreferrer',
+        'data-beacon-article' => '5fbd3942cff47e00160bd248',
+      ],
+      'link'
```

Comment:
Just note that `link` is the default value for the parameter `$linkTag` when the parameter is not specified. So, it would be possible to don't use this parameter in this case. Maybe you know about that, and it's up to you if the value will stay here.